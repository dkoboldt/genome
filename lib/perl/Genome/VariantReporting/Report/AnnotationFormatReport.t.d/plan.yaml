reports:
    'annotation-format':
        interpreters:
            position: {}
            vep: {}
