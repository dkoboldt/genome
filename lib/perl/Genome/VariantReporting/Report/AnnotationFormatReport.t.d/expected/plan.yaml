---
experts: {}
needs_translation: 1
reports:
  annotation-format:
    filters: {}
    interpreters:
      position: {}
      vep: {}
